src/models/inbound/dfspConsentRequests.model.ts

```diff
@@ -85,14 +86,15 @@ export class DFSPConsentRequestsModel
     const { request, toParticipantId } = this.data
 
     try {
-      const response = await this.dfspBackendRequests.validateConsentRequests(request)
+      const response =
+        await this.dfspBackendRequests.validateConsentRequests(request) as BackendValidateConsentRequestsResponse
 
       if (!response) {
-        throw new Errors.MojaloopFSPIOPError('', '', '', Errors.MojaloopApiErrorCodes.TP_FSP_CONSENT_SCOPES_ERROR)
+        throwMojaloopFSPIOPError(Errors.MojaloopApiErrorCodes.TP_CONSENT_REQ_VALIDATION_ERROR)
```

Comment:
when we use `throw` here not from function, it helps TS compiler & linter a lot.
```suggestion
        throw mkMojaloopFSPIOPError(Errors.MojaloopApiErrorCodes.TP_CONSENT_REQ_VALIDATION_ERROR)
```